recipes/s6-init/s6-init/syslogd-srv.filter

```diff
@@ -0,0 +1,3 @@
+#!/bin/sed -f
+s/^\(<[0-9]\{1,3\}>\)\(Jan\|Feb\|Mar\|Apr\|May\|Jun\|Jul\|Aug\|Sep\|Oct\|Nov\|Dec\) [0-9 ]\{2\} [0-9]\{2\}\(:[0-9]\{2\}\)\{2\} /\1/
```

Comment:
they all seem to end up failing someway or the other, but if there is not better solution, fine by me. It's only impacting s6 users anyway